org.horizon_eda.HorizonEDA.json

```diff
@@ -0,0 +1,254 @@
+{
+    "app-id": "org.horizon_eda.HorizonEDA",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "horizon-eda",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--device=dri"
+    ],
+    "modules": [
+        {
+           "name": "mm-common",
+           "cleanup": [ "/" ],
+           "sources": [
+               {
+                   "type": "archive",
+                   "url": "https://download.gnome.org/sources/mm-common/0.9/mm-common-0.9.12.tar.xz",
+                   "sha256": "ceffdcce1e5b52742884c233ec604bf6fded12eea9da077ce7a62c02c87e7c0b"
+               }
+           ]
+        },
+        {
+            "name": "sigc++",
```

Comment:
I assume all these libraries leave some development headers or static libraries around. They should be cleaned up (`cleanup` can be also global).

Reply:
```

Reply:
```
    "cleanup": [
        "*.a",
        "*.la",
        "/include",
        "/lib/*/*.h",
        "/share/oce-0.18",
        "/share/pkgconfig"
    ],
```